src/__internal__/input-icon-toggle/input-icon-toggle.component.js

```diff
@@ -76,7 +76,7 @@ InputIconToggle.propTypes = {
   onBlur: PropTypes.func,
   onMouseDown: PropTypes.func,
   inputIcon: PropTypes.string,
-  size: PropTypes.oneOf(["small", "medium", "large"]),
+  size: PropTypes.oneOf(["extra-small", "small", "medium", "large"]),
```

Comment:
This needs to be updated in the TypeScript interface as well

Reply:
Also `size` is being passed down to `ValidationIcon` and `Icon` which don't have `extra-small` in their prop types so you will need to update those as well